setup.cfg

```diff
@@ -33,7 +33,7 @@ cmdclass =
 install_requires =
   awe_lexica @ git+https://github.com/ArgLab/AWE_Lexica.git
   spacy
-  coreferee
+  coreferee @ git+https://github.com/Arglab/coreferee.git@latest_spacy
```

Comment:
Is this pointing to the appropriate place? Should it be the main branch instead?

What is different about the Arglab coreferee compared to the original corefereee?